src/pages/admin/main/components/statistics-block/components/statistics-metrics-list/StatisticsMetricsList.module.scss

```
@@ -51,33 +51,42 @@
+.raisedValues {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    gap: 2px;
 }
 
-.ua {
+.ua,
+.en {
     @include type.body-2-medium;
     margin: 0;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)